src/docs/user/references/CalendarServiceReference.adoc

```diff
@@ -32,23 +23,31 @@ ProActive user can get this URL from scheduler portal's `Calendar` button
 
 image::calendar_button.png[align="center"]
 
-The URL for the logined user is going to be shown in a popup window
+ProActive user can create a unique private URL by clicking on the `Create` button in the popup window. 
 
-image::calendar_url.png[align="center"]
+image::calendar_url_create.png[align="center"]
 
-By default, only the secured URL is provided, which user can use with Thunderbird, Apple Calendar application with credentials. A private URL can be created and used for Google calendar or Outlook without credentials, the private URL can be regenerated or deleted if it is compromised.
+The generated private URL can then be used in a calendar client application to receive jobs events coming from ProActive Scheduler. The user's private URL can be regenerated or deleted and recreate if it is compromised.
```

Comment:
"The user's private URL can be regenerated, or deleted and recreated if it is compromised." looks better to me